tests/model_hub_tests/transformation_tests/test_pa_transformation.py

```diff
@@ -14,15 +15,28 @@
 def run_pa(tmp_path, model_id, model_link, use_block_indices_inputs, use_score_outputs):
     model = OVModelForCausalLM.from_pretrained(model_id, export=True, trust_remote_code=True)
 
+    before_map = {}
+    for op in model.model.get_ordered_ops():
+        if op.get_type_name() in nodes_to_compare:
+            before_map[op.get_type_name()] = before_map.get(op.get_type_name(), 0) + 1
+
     paged_attention_transformation(model.model, use_block_indices_inputs, use_score_outputs)
 
-    # Test that a _PagedAttentionExtension node appeared after the transformation.
-    pa_counter = 0
+    after_map = {}
     for op in model.model.get_ordered_ops():
-        if isinstance(op, _PagedAttentionExtension):
-            pa_counter += 1
+        if op.get_type_name() in nodes_to_compare:
+            after_map[op.get_type_name()] = after_map.get(op.get_type_name(), 0) + 1
+
+    # Collect the changes of nodes from nodes_to_compare
+    # And check if the numbers correspond to the reference ones
+    resulting_map = {}
+    for op in set(after_map.keys()) | set(before_map.keys()):
+        resulting_map[op] = after_map.get(op, 0) - before_map.get(op, 0)
+
+    use_cache_eviction = use_block_indices_inputs and use_score_outputs
+    reference_map = ref_diff_map_ce[model_id] if use_cache_eviction else ref_diff_map[model_id]
 
-    assert pa_counter > 0, f"The model '{model_id}' has no _PagedAttentionExtension present."
+    assert reference_map == resulting_map
```

Comment:
it's better to print a more verbose message
e.g. 
Unexpected changes were found after applying the SDPAtoPagedAttention transformation.
Parameters Expected: +3 Actual: +2
etc

Reply:
Here python would print a verbose message by itself, no worries here

Reply:
Already a pretty verbose message:

![image](https://github.com/user-attachments/assets/d374b35b-d4d8-41c1-be38-67f7697a2d08)